test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -84,10 +84,12 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 				By("Modifying profile")
 				profile.Spec.WorkloadHints = nil
 
-				profile.Spec.RealTimeKernel = &performancev2.RealTimeKernel{
```

Comment:
I do not see how this can fix the timeout. I see two cases:

1) the RealTimeKernel was false -> both new and old code just keep or set false, no change in the value
2) the RealTimeKernel was true -> both new and old code set false

Then both old and new code call UpdateWithRetry with a profile that asks for regular kernel.

What am I missing here?

Reply:
@MarSik , the timeout was noticed in D/s after we switched the Realtime kernel to false as default in Performance profile.

Reply:
Realtimekernel false was always the default.

But what I do not understand in this test change is the mechanism by which it should fix it. In other words, the added if makes no difference in what gets passed to the updatewithretry method.

`If true: set false` is simply nothing else than `set false`, just mode complicated. It does not change the outcome.

Reply:
According to 0_config (https://github.com/openshift/cluster-node-tuning-operator/blob/master/test/e2e/performanceprofile/functests/0_config/config.go#L195) we set the RealTimeKernel to true when the profile is applied. 

As to why does it timeout when the defaultProfile is false , i agree it should not timeout, this needs to be investigated.

Reply:
Ok, so it was a test default.

But the `if` still makes no difference. Pure write we had there before has the same effect.

Reply:
Agree, the problem of timeout is else where. i will close this PR.